docs/reference/sdl/index.rst

```diff
@@ -82,3 +82,5 @@ to the previous migration:
     functions
```

Comment:
For consistency with the non-reference docs, can we use this ordering?

- annotations
- globals
- acl
- functions
- extensions